backend/resources/org/akvo/dash/migrations_tenants/004-job-execution.up.sql

```diff
@@ -1,11 +1,12 @@
 CREATE TABLE job_execution (
     id text PRIMARY KEY,
-    data_source_id text REFERENCES data_source,
-    started_at timestamptz NOT NULL DEFAULT now(),
-    finished_at timestamptz,
-    error_reason text,
-    created timestamptz DEFAULT now(),
-    modified timestamptz DEFAULT now()
+    data_source_id text REFERENCES data_source ON DELETE CASCADE,
+    dataset_id text REFERENCES dataset ON DELETE CASCADE,
+    type text NOT NULL,
+    log text,
```

Comment:
The idea behind `log` as text, is that we should it could hold just one line (an error reason) or a list of _statements_ from a transformation execution (lines separated by `"\n"`)